Docs v7

namArora3112 opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

<!--- Describe your changes in detail -->

## Related Issue

<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

cacheung commented on README.md:
````diff
@@ -114,180 +114,12 @@ cd ios && pod update && cd ..
-
-```
+Then, initialize the SDK using the following methods:
+- [MobileCore.initializeWithAppId(appId)](https://github.com/adobe/aepsdk-react-native/tree/docs-v7/packages/core#initializewithappid)
````
Use main branch for both links
https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#initializewithappid

namArora3112 replied:
updated

cacheung commented on README.md:
```diff
@@ -114,180 +114,12 @@ cd ios && pod update && cd ..
-
-> For further details on Lifecycle implementation, please refer to the [Lifecycle API documentation](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#lifecycle).
+> [!NOTE]  
+> Starting from Adobe Experience Platform React native **7.x**,  there is no longer a need to initialize the SDK on the [native platforms](https://github.com/adobe/aepsdk-react-native/tree/docs-v7#initializing), as was required in earlier versions.
```
The link here is for the user can refer back to old registering instruction.  The current link doesn't have that info.  
You can create a 6.x branch from Main and point the link to the `initializing` section on  that page.

cacheung commented on README.md:
```diff
@@ -114,180 +114,12 @@ cd ios && pod update && cd ..
-> For further details on Lifecycle implementation, please refer to the [Lifecycle API documentation](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#lifecycle).
+> [!NOTE]  
+> Starting from Adobe Experience Platform React native **7.x**,  there is no longer a need to initialize the SDK on the [native platforms](https://github.com/adobe/aepsdk-react-native/tree/docs-v7#initializing), as was required in earlier versions.
 
```
Add the `Importing the extension` in the main readme to show all the neccessary steps.

namArora3112 replied:
have added that section . can you please review it

cacheung replied:
Let put the `Importing the extension` above of the `Initializing` since we use initialize API after importing the core extension.

## Importing the extension:
In your React Native application, import the extensions.

The following example demonstrates how to import core extensions from the Adobe Experience Platform SDK in your React Native application:

```typescript
import { MobileCore, Lifecycle, Signal, LogLevel, PrivacyStatus, Event } from '@adobe/react-native-aepcore'
```

Refer to each package's documentation for specific import details.

## Initializing

namArora3112 replied:
added

cacheung commented on README.md:
````diff
@@ -114,180 +114,12 @@ cd ios && pod update && cd ..
-@end
-
-```
+Then, initialize the SDK using the following methods:
````
Under `Install AEP npm packages`
Change the below code snippet to version 7.0.0

"dependencies": {
    "react-native": "0.72.5",
    "@adobe/react-native-aepcore": "^6.0.0", //core is required and includes aepcore, aepsignal, aeplifecycle, aepidentity libraries
    "@adobe/react-native-aepedge": "^6.0.0",
    "@adobe/react-native-aepedgeidentity": "^6.0.0",
    "@adobe/react-native-aepedgeconsent": "^6.0.0",
...
},

namArora3112 replied:
updated these versions

cacheung commented:
Don't forget to update the `Migration` doc and `Expo` doc.  Not necessary to update these doc before the release.

namArora3112 commented on README.md:
```diff
@@ -114,181 +114,57 @@ cd ios && pod update && cd ..
-    })
-  }
+> [!NOTE]  
+> Starting from Adobe Experience Platform React native **7.x**,  there is no longer a need to initialize the SDK on the [native platforms](https://github.com/adobe/aepsdk-react-native/tree/main#initializing), as was required in earlier versions.
```
link updated

cacheung commented:
@namArora3112 
Can you check each package, to make sure they all have 
npm and yarn information?
I saw some packages do not have `yarn add @adobe/react-native-aepxxx` info

Also make sure 
Initializing with SDK:
Importing the extension:
have the same header level in all the packages

namArora3112 commented:
> @namArora3112 Can you check each package, to make sure they all have npm and yarn information? I saw some packages do not have `yarn add @adobe/react-native-aepxxx` info
> 
> Also make sure Initializing with SDK: Importing the extension: have the same header level in all the packages

Have addressed your concerns

cacheung commented on packages/edgeidentity/README.md:
```diff
@@ -17,84 +17,27 @@ See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native
 
+Refer to the root [Readme](https://github.com/adobe/aepsdk-react-native/blob/main/README.md) for more information about the SDK setup.
 
 ### [Identity for Edge Network](https://developer.adobe.com/client-sdks/documentation/identity-for-edge-network)
```
We can remove this, we have it at the top of the readme.

namArora3112 replied:
have remove it

cacheung commented on packages/messaging/README.md:
```diff
@@ -35,77 +35,13 @@ yarn add @adobe/react-native-aepmessaging
 
 ### [Messaging](https://developer.adobe.com/client-sdks/documentation/adobe-journey-optimizer)
```
We can remove this, we have it at the top of the readme.

namArora3112 replied:
have removed it
